plugins/rikki/loungeviews/components/playoffoverview/default.htm

```diff
@@ -100,8 +100,8 @@ <h6>Team Description:</h6>
                     {% partial '@divisionView' div=division %}
                 </div>
                 {% endfor %}
-                
 
+                <div>All times in {{__SELF__.timezone}} {{__SELF__.timezoneOffset}}</div>
```

Comment:
Here the timezone is moved up to the top of the tab rather than being with every displayed time